masonry/src/widget/textbox.rs

```diff
@@ -259,6 +259,7 @@ impl Widget for Textbox {
             LifeCycle::BuildFocusChain => {
                 ctx.register_for_focus();
                 // TODO: This will always be empty
+                #[cfg(FALSE)]
                 if !self.editor.text().links().is_empty() {
                     tracing::warn!("Links present in text, but not yet integrated");
                 }
```

Comment:
Probably should just have the same comment as `Prose`?

Reply:
I'm removing references to links for now. They were copy-pasted from Label's implementation, but I don't see links in editable text being implemented any time soon.